src/test/node/node-runtime.test.ts

```diff
@@ -47,6 +48,22 @@ describe('node runtime', () => {
     handle.assertLog({ substring: true });
   });
 
+  itIntegrates('skipFiles skip node internals', async ({ r }) => {
+    await r.initialize;
+    const cwd = join(testWorkspace, 'simpleNode');
+    const handle = await r.runScript(join(cwd, 'index.js'), { skipFiles: ['<node_internals>/**'] });
+    await handle.dap.setBreakpoints({
+      source: { path: join(cwd, 'index.js') },
+      breakpoints: [{ line: 1, column: 1 }],
+    });
+
+    handle.load();
+    const stoppedParams = await handle.dap.once('stopped');
+    await sleep(200);
```

Comment:
This delay shouldn't be necessary, were you hitting issues here?

Reply:
ah, I see the comment on the similar sleep below. Maybe comment this line too. Alternatively, maybe a better approach, should the client requesting a stacktrace caused any debounced blackboxing to apply immedately? If the client requests a stacktrace quickly after launch the user would see the lack of blackboxing within their ui.

Reply:
It's too late at that point - the blackbox patterns have not actually been sent to the runtime, and the stacktrace hints should match what the runtime knows.

Reply:
mm, that's unfortunate

Reply:
added comments